src/python/WMCore/MicroService/MSTransferor/RequestInfo.py

```diff
@@ -72,6 +70,33 @@ def __call__(self, reqRecords):
 
         return workflows
 
+    def classifyWorkflows(self, reqRecords):
+        """
+        This method classifies input workflows, making it easier
```

Comment:
What does "input workflow" mean in this context? My feeling is that this method classifies all workflows.